packages/fxa-admin-panel/server/config/index.ts

```diff
@@ -32,6 +32,33 @@ const conf = convict({
     // Note: This format is a number because the value needs to be in seconds
     format: Number,
   },
+  csp: {
+    enabled: {
+      default: true,
+      doc: 'Send "Content-Security-Policy" header',
+      env: 'CSP_ENABLED',
+    },
+    reportUri: {
+      default: 'https://accounts.firefox.com/_/csp-violation',
+      doc: 'Location of "report-uri" for full, blocking CSP rules',
```

Comment:
Hmm so when this was setup in payments we decided to just forward CSP violations to accounts.firefox.com, but I'm not sure if we want to do that with the admin panel. We probably want to remove `https://accounts.firefox.com` from lines 42 and 52 (check out the content server config) but someone else may need to chime in here. 🤔

Reply:
So, it turns out a couple things are relevent here. jbuck helped me figure it out.

Further to our discussion around whether or not reports are actually handled, it turns out they are indeed. The paths `/_/csp-violation` and `/_/csp-violation-report-only` are handled at the nginx level.

Then, in production the vars for `CSP_REPORT_URI` and `CSP_REPORT_ONLY_URI` are set so the values here don't matter too much. I was advised that setting them to the above paths are fine in this case.